inspire_crawler/tasks.py

```diff
@@ -108,16 +133,27 @@ def _extract_results_data(results_path):
     if results_data is None:
         results_data = _extract_results_data(results_path)
 
-    for record in results_data:
+    for result in results_data:
+        crawl_result = _check_crawl_result_format(result)
```

Comment:
hmm.... I'd rename the `_check_crawl_result_format`, anything that starts with `check` is not expected to return anything (but raise exception/log error), for example:
```python

def check_is_bool(maybe_bool):
    if not isinstance(bool, maybe_bool):
        raise Exception("The value '%s' is not a bool" % maybe_bool)

def is_bool(maybe_bool):
    return isinstance(bool, maybe_bool)

def to_bool(maybe_bool):
    if isinstance(bool, maybe_bool):
        return maybe_bool
    else:
        return bool(maybe_bool)
```

That way when you use them you do:
```python
try:
    check_bool(mybool)
except Exception:
    return 'it was not a bool'

if is_bool(mybool):
    return 'it was not a bool'

return to_bool(mybool)
```

Reply:
I think that that function does too much stuff....
Maybe split into two, something like `check_format` and `bad_formatted_crawler_result_to_crawler_result_error` ....  damn naming.... xd

Reply:
I'll make it more beautiful :man_dancing: